Makefile

```diff
@@ -0,0 +1,58 @@
+export PATH := $(abspath bin/):${PATH}
+
+FORMATTING_BEGIN_YELLOW = \033[0;33m
+FORMATTING_BEGIN_BLUE = \033[36m
+FORMATTING_END = \033[0m
+
+TESTS_TIMEOUT="15m"
+FOCUS=""
+
+help:
+	@printf -- "\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ██████╗░███████╗░█████╗░██╗░░██╗██╗░░██╗░█████╗░██╗░░░██╗░██████╗███████╗${FORMATTING_END}\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ██╔══██╗██╔════╝██╔══██╗██║░██╔╝██║░░██║██╔══██╗██║░░░██║██╔════╝██╔════╝${FORMATTING_END}\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ██║░░██║█████╗░░██║░░╚═╝█████═╝░███████║██║░░██║██║░░░██║╚█████╗░█████╗░░${FORMATTING_END}\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ██║░░██║██╔══╝░░██║░░██╗██╔═██╗░██╔══██║██║░░██║██║░░░██║░╚═══██╗██╔══╝░░${FORMATTING_END}\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ██████╔╝███████╗╚█████╔╝██║░╚██╗██║░░██║╚█████╔╝╚██████╔╝██████╔╝███████╗${FORMATTING_END}\n"
+	@printf -- "${FORMATTING_BEGIN_BLUE}     ╚═════╝░╚══════╝░╚════╝░╚═╝░░╚═╝╚═╝░░╚═╝░╚════╝░░╚═════╝░╚═════╝░╚══════╝${FORMATTING_END}\n"
+	@printf -- "\n"
+	@printf -- "-----------------------------------------------------------------------------------\n"
+	@printf -- "\n"
+	@awk 'BEGIN {FS = ":.*##"; printf "Usage: make ${FORMATTING_BEGIN_YELLOW}<target>${FORMATTING_END}\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  ${FORMATTING_BEGIN_BLUE}%-46s${FORMATTING_END} %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+
+GOLANGCI_VERSION = 1.42.0
+TESTS_TIMEOUT="15m"
+
+##@ Tests
+
+.PHONY: tests-modules tests-matrix tests-openapi
+tests-modules: ## Run unit tests for modules hooks and templates.
+	@go test -timeout=${TESTS_TIMEOUT} -vet=off ./modules/... ./global-hooks/... ./ee/modules/... ./ee/fe/modules/...
+
+tests-matrix: ## Test how helm templates are rendered with different input values generated from values examples. Use 'FOCUS' environment variable to run tests for a particular module.
```

Comment:
Great work!

Can I add two small things? We need two commands:
`run-all` - run all checks for all modules.
`run-all-for-module` - run all checks for one module. Matrix tests and hooks tests for all modules can be slow, but we often work with one module and we do not want run tests for all modules.

What do you think about it?

Reply:
Yeah, but not in the scope of this PR.

Reply:
Ok